cmd/genji/shell/command.go

```diff
@@ -92,16 +165,100 @@ func runIndexesCmd(db *genji.DB, in []string) error {
 	return fmt.Errorf("usage: .indexes [tablename]")
 }
 
-// runHelpCmd display all available dot commands.
-func runHelpCmd() error {
-	var keys []string
-	for k := range commands {
-		keys = append(keys, k)
+// displayHelp show all available commands.
+func displayHelp(db *genji.DB, cmd []string) error {
+	for _, c := range commands.cmds {
+		indent := 25 - len(c.getOptions()) - len(c.getName())
```

Comment:
What does 25 mean?

Reply:
it's the initial number of spaces to keep all descriptions aligned when they are displayed.